lib/formatters.js

```diff
@@ -1,27 +1,50 @@
 'use strict';
 
 const Promise = require('bluebird');
+const { DataEntity } = require('@terascope/utils');
 
 // No parsing, leaving to reader or a downstream op.
-function pass(data) {
-    return data;
+function raw(data, logger, metadata, opConfig) {
```

Comment:
metadata should be last since it may be optional but opConfig is required.

```suggestion
function raw(data, logger, opConfig, metadata) {
```

Reply:
fixed